.github/workflows/publish.yml

```diff
@@ -4,6 +4,16 @@ jobs:
   Publish:
     runs-on: ubuntu-latest
     steps:
+      - uses: actions/checkout@v2
```

Comment:
I totally missed this aspect of the pypi-publish action.

https://github.com/marketplace/actions/pypi-publish

> This GitHub Action has nothing to do with building package distributions. Users are responsible for preparing dists for upload by putting them into the dist/ folder prior to running this Action.